eva/udfs/saliency_feature_extractor.py

```diff
@@ -0,0 +1,83 @@
+# coding=utf-8
+# Copyright 2018-2022 EVA
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import cv2
+import kornia
+import numpy as np
+import pandas as pd
+import torch
+import torchvision
+import torch.nn as nn
+import torch.nn.functional as F
+
+from eva.catalog.catalog_type import NdArrayType
+from eva.udfs.abstract.abstract_udf import AbstractUDF
+from eva.udfs.decorators.decorators import forward, setup
+from eva.udfs.decorators.io_descriptors.data_types import PandasDataframe
+from eva.udfs.gpu_compatible import GPUCompatible
+from torchvision.transforms import Compose, ToTensor, Resize
+from PIL import Image
```

Comment:
make sure you had successfully executed `bash script/test/test.sh -m "LINTER"   `